lib/modules/manager/gitlabci/extract.ts

```diff
@@ -146,15 +146,15 @@ export async function extractAllPackageFiles(
     if (is.array(doc?.include)) {
       for (const includeObj of doc.include) {
         if (is.string(includeObj.local)) {
-          const fileObj = includeObj.local.replace(regEx(/^\//), '');
+          const fileObj = includeObj.local.replace(regEx(/^\/+/), '');
```

Comment:
use trim leading slash function from url util?

Reply:
It only trims one :)

Reply:
i know, see my other comment, it should trim all, like the trailing variant

Reply:
OK, so let's improve trimLeadingSlash to replace multiple and use that function instead.